### PR TITLE
docs(ootbc): added Blue Prism connector doc

### PR DESCRIPTION
**Pull request by igpetrov**
## What is the purpose of the change

Adding a new page for the connectors related to a Blue Prism RPA connector.

## Are there related marketing activities

TBD.

## When should this change go live?

TBD - currently on hold but ready for review.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by igpetrov** [comment]
Hi @bastiankoerber. Could you please review this BP docs and give your suggestions?

**Comment by akeller** [comment]
Looks like this PR has some conflicts. @igpetrov do you have time to clean this up before next week? @bastiankoerber can hopefully review before his Community Summit workshop and talk.

**Comment by akeller** [comment]
Makes sense to me @igpetrov. Thanks for clarifying so quickly!

**Comment by igpetrov** [comment]
Hi @christinaausley,

This one is ready for review but please don't merge it yet, just waiting for a final approval.

**Comment by igpetrov** [comment]
Hi @christinaausley, this one is ready to be reviewed and merged :)

Thank you!

**Review by christinaausley — APPROVED** [review]
One non-blocking comment, and technical review in latest commit 👍

File: docs/components/connectors/out-of-the-box-connectors/blueprism.md
Comment (igpetrov) [reply docs/components/connectors/out-of-the-box-connectors/blueprism.md]: Totally makes sense. Removed it.

**Review by christinaausley — APPROVED** [review]

**Review by christinaausley — DISMISSED** [review]

**Comment by christinaausley** [comment]
Did you want @bastiankoerber to have a look at this? If not, feel free to squash and merge.

**Review by bastiankoerber — APPROVED** [review]
looks good to me

**Pull request merged**
